lib/classes/squad.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:collection/collection.dart';
+import 'package:phone_factory/classes/squad_members/squad_member.dart';
+
+class Squad {
+  int _captain = 1;
+  static const int _initSquadMemberCount = 3;
+  List<SquadMember> _squad = [];
+  int _antidotCount = 1;
+  List<SquadMember> get squad => _squad;
+
+  set squad(List<SquadMember> value) {
+    _squad = value;
+  }
+
+  Squad() {
+    _fillSquad();
+  }
+
+  List<SquadMember> get squadMembersCount => _squad;
+  int get antidotCount => _antidotCount;
+  int get capatin => _captain;
+
+  set captain(int newValue) {
+    if ((_captain - newValue) >= 0) {
+      _captain = newValue;
+    }
+  }
+
+  void addAntidot() {
+    _antidotCount += 1;
+  }
+
+  void _fillSquad() {
+    for (var i = 0; i < _initSquadMemberCount; i++) {
+      _squad.add(SquadMember());
+    }
+  }
+
+  void wasInfected() {
+    SquadMember? helsyMember = _squad.firstWhereOrNull((member) => member.isInfected == false);
+    if (helsyMember == null) {
+      print('Нема здоровых членів команди. Перевір Склад команди');
+    } else {
+      helsyMember.isInfected = true;
+      print('Увага!!!Один із єкіпажу інфікований!!!');
+    }
+  }
+
+  bool isMemberMutant() {
```

Comment:
this function needs to return true of false. please, move all extra logic outside this function